src/lib/cyleaflet_clientside.js

```diff
@@ -0,0 +1,88 @@
+import proj4 from 'proj4'
+
+
+if (!window.dash_clientside) {
+    window.dash_clientside = {};
+}
+
+window.dash_clientside.cyleaflet_utils = {
```

Comment:
Nonblocking, but do these utils need to be exposed to the end user or do they just get used inside `dash_clientside.cyleaflet`? If the latter, they could just be local functions in this file (and `transformElements` could be inlined in `dash_clientside.cyleaflet`)

Reply:
Good point -- changed all of these to local variables.